lib/storage_api/src/replay_wire_format/v1.rs

```diff
@@ -0,0 +1,74 @@
+//! We need to not accidentally change the replay wire format
+//! but there is no way in Rust to get a stable unique ID for a type,
+//! so instead we define it in this separate file.
+//!
+//! Do not change this file under any circumstances. Copy it instead. May be deleted when obsolete.
+//! (This is enforced by CI)
+
+use bincode::{Decode, Encode};
+
+// It is somewhat safe to assume that these will not change
+use alloy::primitives::{B256, U256, ruint::aliases::B160};
+
+/// The format ReplayRecords are currently sent in
+#[derive(Encode, Decode)]
+pub struct ReplayWireFormat {
```

Comment:
I would call it `ReplayWireFormatV1` to be more explicit.. Otherwise we'd had files in different modules but with the same name - confusing?